python/eos/analysis_TEST.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+import eos
+import numpy as np
+
+class ClassMethodTests(unittest.TestCase):
+
+    def test_parameter_scaling(self):
+
+        analysis_args = {
+            'global_options': { 'form-factors': 'BSZ2015', 'model': 'CKMScan' },
+            'priors': [
+                { 'parameter': 'CKM::abs(V_cb)',           'min':  38e-3, 'max':  45e-3 , 'type': 'uniform'},
+                { 'parameter': 'B->D::alpha^f+_0@BSZ2015', 'min':  0.0,   'max':  1.0   , 'type': 'uniform'},
+                { 'parameter': 'B->D::alpha^f+_1@BSZ2015', 'min': -4.0,   'max': -1.0   , 'type': 'uniform'},
+                { 'parameter': 'B->D::alpha^f+_2@BSZ2015', 'min': +4.0,   'max': +6.0   , 'type': 'uniform'},
+                { 'parameter': 'B->D::alpha^f0_1@BSZ2015', 'min': -1.0,   'max': +2.0   , 'type': 'uniform'},
+                { 'parameter': 'B->D::alpha^f0_2@BSZ2015', 'min': -2.0,   'max':  0.0   , 'type': 'uniform'}
+            ],
+            'likelihood': [
+                'B->D::f_++f_0@HPQCD2015A',
+                'B->D::f_++f_0@FNALMILC2015B',
+                'B^0->D^+e^-nu::BRs@Belle-2015A',
+                'B^0->D^+mu^-nu::BRs@Belle-2015A'
+            ]
+        }
+
+        # Test analysis definition
+        analysis = eos.Analysis(**analysis_args)
```

Comment:
This call fails in our CI pipeline. Ideas?

Reply:
I do know ^^
This is weird... Are we sure `make install` is ran before we test?

Reply:
@dvandyk should be able to settle this quickly :)

I think we don't install, but set the paths to the build dirs. Not sure if these are correct or if it's something else.

Reply:
Indeed `make install` is not the culprit, it is locally not even needed... so I don't know yet